app/gui/src/model/execution_context.rs

```diff
@@ -216,6 +216,78 @@ pub struct LocalCall {
 
 
 
+// ==============================
+// === QualifiedMethodPointer ===
+// ==============================
+
+/// A method pointer containing the qualified module and type names.
+#[derive(Clone, Debug, PartialEq)]
+pub struct QualifiedMethodPointer {
+    /// A module name containing the method.
+    pub module:          module::QualifiedName,
+    /// A type on which the method is defined.
+    pub defined_on_type: module::QualifiedName,
```

Comment:
If this is a type, why it uses `module::QualifiedName` rather than `tp::QualifiedName`?